internal/pkg/manifest/workload.go

```diff
@@ -37,7 +37,8 @@ type Workload struct {
 
 // Image represents the workload's container image.
 type Image struct {
-	Build BuildArgsOrString `yaml:"build"` // Path to the Dockerfile.
+	Build    BuildArgsOrString `yaml:"build"` // Path to the Dockerfile.
+	Location *string           `yaml:"location"`
```

Comment:
Can we update the comments here:
```suggestion
	Build    BuildArgsOrString `yaml:"build"` // Build an image from a Dockerfile.
	Location *string           `yaml:"location"`  // Use an existing image instead.
```